e2e/libs/recurring_job/crd.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from recurring_job.base import Base
+from recurring_job.rest import Rest
+from kubernetes import client
+
+class CRD(Base):
+
+    def __init__(self):
+        self.rest = Rest()
+
+    def create(self, name, task, groups, cron, retain, concurrency, label):
+        logging.warn("NotImplemented")
```

Comment:
Why need log here? Is it because we don't have a CRD implementation but rather delegate to the API?

You probably can just have `logging.warn("Delegating the create call to API because there is no CRD implementation").`

Reply:
All the print/logging things don't matter since the desirable way to get useful runtime information is still investigated.